### PR TITLE
AT_14.06._002 | Rename Multi-configuration project

### PR DESCRIPTION
**Pull request by TatianaBM**
https://trello.com/c/N0tOjTJs/853-at1406002-rename-multi-configuration-projectpositive

### CODE REVIEW

**Review by matvlesia — APPROVED** [review]

File: cypress/e2e/multiConfigurationProjectRename.cy.js [hunk 1]
```diff
@@ -21,4 +24,17 @@ describe('multiConfigurationProjectRename', () => {
         cy.get('.jenkins-breadcrumbs__list-item').contains('Dashboard').click()
         })
     })
+
+    it.only('AT_14.06._002 | Rename Multi-configuration project_Positive', () => {
```
Comment (elviraKh1) [inline cypress/e2e/multiConfigurationProjectRename.cy.js:28]: you shoud remove only

Comment (TatianaBM) [reply cypress/e2e/multiConfigurationProjectRename.cy.js:28]: thank u 
completely forgot

File: cypress/e2e/multiConfigurationProjectRename.cy.js [hunk 2]
```diff
@@ -21,4 +24,17 @@ describe('multiConfigurationProjectRename', () => {
         cy.get('.jenkins-breadcrumbs__list-item').contains('Dashboard').click()
         })
     })
+
+    it('AT_14.06._002 | Rename Multi-configuration project_Positive', () => {
+        let renameValidName = 'RenamedProject123'
```
Comment (Rom-T) [inline cypress/e2e/multiConfigurationProjectRename.cy.js:29]: Please move `'RenamedProject123'` to the fixtures

File: cypress/e2e/multiConfigurationProjectRename.cy.js [hunk 3]
```diff
@@ -21,4 +24,17 @@ describe('multiConfigurationProjectRename', () => {
...
+        cy.get('td:nth-child(3) a[href^="job/"]').realHover()
+        cy.get('.jenkins-table__link button.jenkins-menu-dropdown-chevron').click()
+        cy.get('.first-of-type > li').contains('Rename').click()
+        cy.get('input[name="newName"]')
+            .clear()
+            .type(`${renameValidName}{enter}`)
+        cy.get('h1.page-headline')
+            .should('include.text', 'Project')
+            .and('include.text', renameValidName)
```
Comment (Rom-T) [inline cypress/e2e/multiConfigurationProjectRename.cy.js:38]: And why 2 asserts for one element?

File: cypress/e2e/multiConfigurationProjectRename.cy.js (hunk 2 above)
Comment (TatianaBM) [reply cypress/e2e/multiConfigurationProjectRename.cy.js]: thank u Roman, added to fixtures

File: cypress/e2e/multiConfigurationProjectRename.cy.js (hunk 3 above)
Comment (TatianaBM) [reply cypress/e2e/multiConfigurationProjectRename.cy.js]: thank u , last assert is removed

**Review by OLgaByR — APPROVED** [review]

**Pull request merged**